blocks/components/editable/index.js

```diff
@@ -57,6 +59,28 @@ export default class Editable extends wp.element.Component {
 		this.props.onChange( value );
 	}
 
+	onKeyDown( event ) {
+		if ( ! this.props.tagName && event.keyCode === 13 ) {
+			// Wait for the event to propagate
+			setTimeout( () => {
+				// Getting the content before and after the cursor
+				this.editor.selection.getStart();
+				const childNodes = Array.from( this.editor.getBody().childNodes );
+				const splitIndex = childNodes.indexOf( this.editor.selection.getStart() );
+				const getHtml = ( nodes ) => nodes.reduce( ( memo, node ) => memo + node.outerHTML, '' );
+				const beforeNodes = childNodes.slice( 0, splitIndex );
+				const before = getHtml( beforeNodes );
+				const after = getHtml( childNodes.slice( splitIndex ) );
+
+				// Splitting into two blocks
+				this.editor.setContent( this.props.value );
```

Comment:
I'm confused by this line. Wouldn't this reset the content back to its original value prior to the split? I don't see that in practice, but confused as to why it wouldn't.